src/main/java/frc/robot/commands/ArmCommand.java

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import com.revrobotics.AbsoluteEncoder;
+import com.revrobotics.SparkMaxAbsoluteEncoder;
+import com.revrobotics.CANSparkMax.ControlType;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.ArmSubsystem;
+import frc.robot.subsystems.SwerveSubsystem;
+
+public class ArmCommand extends CommandBase {
+  double kP = 0;
+  double kI = 0;
+  double kD = 0;
+  double integralMinimum = -0.5;
+  double integralMaximum = 0.5;
+
+  PIDController pid = new PIDController(kP, kI, kD);
```

Comment:
ArmCommand does not need the pid controller. Arm system has two pid controllers for top and bottom pivots.